### PR TITLE
Update Readme to include dependency list

### PR DESCRIPTION
**Pull request by maxwell**
Added a section for the Python dependencies needed to run ```print-battleground-state-changes```

### CODE REVIEW

**Comment by titanous** [comment]
This should just suggest `pip install -r requirements.txt`.

**Comment by maxwell** [comment]
Nice to hear from you Jonathan.  I've made the update as you've suggested.

File: README.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ Simon Willison: <https://simonwillison.net/2020/Oct/9/git-scraping/>
 
 ## Development
 
+Dependencies
+
+* Python 3 + is required
```
Comment (titanous) [inline README.md]:
```suggestion
* Python 3 is required
```

**Review by titanous — CHANGES_REQUESTED** [review]

**Comment by titanous** [comment]
Thanks!

**Pull request merged**